pom.xml

```diff
@@ -0,0 +1,703 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <!-- Model Information -->
+  <modelVersion>4.0.0</modelVersion>
+
+  <!-- Artifact Information  -->
+  <groupId>xyz.block</groupId>
+  <artifactId>web5-parent</artifactId>
+  <version>0.0.0-main-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>Web5 SDK for the JVM</name>
+  <url>https://developer.tbd.website</url>
+  <description>Build Aggregator and Parent for Web5 SDK for the JVM</description>
+  <inceptionYear>2023</inceptionYear>
+
+  <!-- SCM -->
+  <scm>
+    <connection>scm:git:git://github.com/TBD54566975/web5-kt.git</connection>
+    <!-- This has to be HTTPS, not git://, for maven-release-plugin to do AUTH correctly -->
+    <developerConnection>scm:git:https://github.com/TBD54566975/web5-kt.git</developerConnection>
+    <url>https://github.com/TBD54566975/web5-kt</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <!-- Developers -->
+  <developers>
+    <developer>
+      <id>TBD54566975</id>
+      <name>Block, Inc.</name>
+      <email>releases@tbd.email</email>
+    </developer>
+  </developers>
+
+  <!-- Issues -->
+  <issueManagement>
+    <system>github</system>
+    <url>https://github.com/TBD54566975/web5-kt/issues</url>
+  </issueManagement>
+
+  <!-- Licenses -->
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <!-- Aggregate Modules -->
+  <modules>
+    <module>common</module>
+    <module>credentials</module>
+    <module>crypto</module>
+    <module>dids</module>
+    <module>distribution</module>
+    <module>jose</module>
+    <module>testing</module>
```

Comment:
what does this block do? i'm hoping this doesn't mean that testing is bundled in the web5 package.

Reply:
This tells the parent POM which builds to run. 

The rollup you're looking for is the `distribution` module, which then in turn defines the deps in the rollup:
https://github.com/TBD54566975/web5-kt/blob/issue-217/maven-build/distribution/pom.xml#L33

Reply:
`<module>` tells Maven which submodules are part of the main build.